driverinference/README.md

```diff
@@ -0,0 +1,306 @@
+# Inference Scaling Test
+
+SmartSim-Scaling offers two inference versions:
+
+ 1. Inference Standard              (C++ client and SmartRedis Orchestrator)
+ 2. Inference Co-located Python     (C++ client and SmartRedis Orchestrator)
+
+Continue below for more information on both respective tests.
+
+## Client Description
+
+The inference tests run as an MPI program where a single SmartRedis C++ client
+is initialized on every rank.
+
+Currently supported inference tests
+
+  1) Resnet50 CNN with ImageNet dataset
+
+Each client performs 101 executions of the following commands. The first iteration is a warmup;
+the next 100 are measured for inference throughput. 
+
+  1) ``put_tensor``     (send image to database)
+  2) ``run_script``     (preprocess image)
+  3) ``run_model``      (run resnet50 on the image)
+  4) ``unpack_tensor``  (Retrieve the inference result)
+
+The input parameters to the test are used to generate permutations
+of tests with varying configurations.
+
+## The model
+As Neural Network, we use Pytorch's implementation of Resnet50. The script `imagenet/model_saver.py`
+can be used to generate the model for CPU or GPU. By navigating to the `imagenet` folder, the CPU model
+can be created running 
+
+```bash
+python model_saver.py
+```
+
+and the GPU model can be created running
+
+```bash
+python model_saver.py --device=GPU
+```
```

Comment:
Add a note here that the node on which this command is run should have a GPU or it will fail